plugins/generator-1.20.1/forge-1.20.1/templates/gui/gui_window.java.ftl

```diff
@@ -116,20 +116,20 @@ public class ${name}Screen extends AbstractContainerScreen<${name}Menu> {
 		</#list>
 	}
 
-	@Override protected void renderBg(PoseStack ms, float partialTicks, int gx, int gy) {
+	@Override protected void renderBg(GuiGraphics guiGraphics, float partialTicks, int gx, int gy) {
 		RenderSystem.setShaderColor(1, 1, 1, 1);
 		RenderSystem.enableBlend();
 		RenderSystem.defaultBlendFunc();
 
 		<#if data.renderBgLayer>
 			RenderSystem.setShaderTexture(0, texture);

```

Comment:
I think the setShaderTexture calls are not needed anymore since blit handles this now

Reply:
Example: https://github.com/Pylo/MCreatorLink/commit/14ade07b7a4085fea2bc1f4a66229f6bf834c98e#diff-1f9614392f854f5f5b957a08f20e1e9453e0e0ff1e29b17bc55acc645c2f6766